bricksrc/equipment.py

```diff
@@ -179,6 +179,7 @@
     },
     "Elevator": {"tags": [TAG.Elevator, TAG.Equipment]},
     "Security_Equipment": {"tags": [TAG.Security, TAG.Equipment]},
+    "Camera_Equipment": {"tags": [TAG.Camera, TAG.Equipment]},
```

Comment:
@epaulson We don't have to say it's `Camera_Equpiment` though we need to add that in the Tag level. Could you change this to like this?: `"Camera": {"tags": [TAG.Camera, TAG.Equipment]},`

Reply:
Done